cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -29,7 +29,7 @@ include::modules/op-constructing-pipelines-using-pipeline-builder.adoc[leveloffs
 
 == Creating OpenShift Pipelines along with applications
 
-To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. 
+To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image.
```

Comment:
This content is good, but it does not appear in the screenshot you provided of the preview content: 
<img width="1090" alt="Screen Shot 2022-07-14 at 10 49 16 AM" src="https://user-images.githubusercontent.com/70717303/179011109-3bb28422-51c3-43b9-9cf9-885e8252021b.png">

Reply:
So even though I think the content is good, I'm concerned why it's not included in the preview version of the content.

Reply:
> So even though I think the content is good, I'm concerned why it's not included in the preview version of the content.

@jc-berger, The content you see is already published on the portal, i did not change anything. I guess i might have removed a space in the end because of which you see this content here.

![image](https://user-images.githubusercontent.com/94683525/179154201-5f5dd143-88b3-46f3-ab7f-890c09ed76aa.png)

Reply:
FYI, the dark red spot shows that a extra space was removed after the period in line 32.